munimap/translations/de/LC_MESSAGES/messages.po

```diff
@@ -7,28 +7,28 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PROJECT VERSION\n"
 "Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
-"POT-Creation-Date: 2021-07-15 11:01+0200\n"
+"POT-Creation-Date: 2022-05-24 18:24+0200\n"
```

Comment:
Is the binary file `munimap/translations/de/LC_MESSAGES/messages.po` meant to be committed?

Reply:
This file can already be found in the repository